### PR TITLE
Adding scaler function and test

### PR DESCRIPTION
**Pull request by camadi**
First test file for scaler function

### CODE REVIEW

File: PrepPy/scaler.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    if not isinstance(colnames, list):
+        raise Exception('Numeric column names is not a list')
+    
+    if X_train.empty == True:
+        raise Exception('X_train cannot be empty')
+    
+    if X_test.empty == True:
+        raise Exception('X_test cannot be empty')
+        
+
```
Comment (matthewconnell) [inline PrepPy/scaler.py]: This seems like a lot of if statements. Is there any way to make this shorter?

File: tests/test_scaler.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+ 
+def test_output(input_data):
+    #Test data
+    X_train = input_data['X_train']
+    X_test = input_data['X_test']
+    X_validation = input_data['X_validation']
+    colnames = input_data['colnames']
+    X_train_empty = input_data['X_train_empty']
+    assert np.isclose(scaler.scaler(X_train, X_validation, X_test, colnames)['X_train']['usage'][0], -1.135549947915338) == True
+    assert np.isclose(scaler.scaler(X_train, X_validation, X_test, colnames)['X_test']['usage'][2], -1.3728129459672882) == True
```
Comment (matthewconnell) [inline tests/test_scaler.py]: Why were those values chosen?

File: PrepPy/scaler.py (hunk 1 above)
Comment (camadi) [reply PrepPy/scaler.py]: Each if statement checks that only valid data types are accepted, lest the function throws exception prior to running

File: tests/test_scaler.py (hunk 2 above)
Comment (camadi) [reply tests/test_scaler.py]: Those are the expected values when the function runs the helper data

**Pull request merged**